addons/im_livechat/static/src/core/web/discuss_sidebar_categories_livechat.js

```diff
@@ -5,7 +5,10 @@ import { discussSidebarCategoriesRegistry } from "@mail/discuss/core/web/discuss
 discussSidebarCategoriesRegistry.add(
     "livechats",
     {
-        predicate: (store) => store.discuss.livechat.threads.some((thread) => thread?.is_pinned),
+        predicate: (store) =>
+            store.discuss.livechat.threads.some(
+                (thread) => thread?.displayToSelf || thread?.eq(store.discuss.thread)
```

Comment:
It's not very good functionally that the thread disappears after clicking on another thread. e.g. user opens a live chat history, goes to another chat to answer a message, the previous chat history disappears: he now has to go back to the live chat app/find the session/re-open in discuss.